README.md

```diff
@@ -61,54 +61,69 @@ PM> Install-Package Neo4j.Driver.Signed
 Connect to a Neo4j database
 
 ```csharp
-IDriver driver = GraphDatabase.Driver("neo4j://localhost:7687", AuthTokens.Basic("username", "pasSW0rd"));
-IAsyncSession session = driver.AsyncSession(o => o.WithDatabase("neo4j"));
-try
+using (var driver = GraphDatabase.Driver("bolt://localhost:7687", AuthTokens.Basic("neo4j", "password")))
 {
-    IResultCursor cursor = await session.RunAsync("CREATE (n) RETURN n");
-    await cursor.ConsumeAsync();
+    var queryOperation = await driver.ExecutableQuery("CREATE (n) RETURN n").ExecuteAsync();
 }
-finally
-{
-    await session.CloseAsync();
-}
-
-...
-await driver.CloseAsync();
 ```
 
 There are a few points that need to be highlighted when adding this driver into your project:
 
-* Each `IDriver` instance maintains a pool of connections inside, as a result, it is recommended to only use **one
-  driver per application**.
-* It is considerably cheap to create new sessions and transactions, as sessions and transactions do not create new
-  connections as long as there are free connections available in the connection pool.
+* Each `IDriver` instance maintains a pool of connections inside; as a result, it is recommended that you use **only one driver per application**.
+* It is considerably cheap to create new sessions and transactions, as sessions and transactions do not create new  connections as long as there are free connections available in the connection pool.
 * The driver is thread-safe, while the session or the transaction is not thread-safe.
 
 ### Parsing Result Values
 
-#### Record Stream
+#### Query Execution Result
 
-A cypher execution result is comprised of a stream records followed by a result summary.
-The records inside the result are accessible via `FetchAsync` and `Current` methods on `IResultCursor`.
-Our recommended way to access these result records is to make use of methods provided by `ResultCursorExtensions` such
-as `SingleAsync`, `ToListAsync`, and `ForEachAsync`.
+The result of executing a query in the way shown above is an `EagerResult<T>`, where `T` is the type of data returned from the query. In the simplest case, this will be an `IReadOnlyList<IRecord>`, which is a fully materialised list of the records returned by the query. Other types of results can be used by using the fluent API exposed by the `IExecutableQuery` type. 
```

Comment:
I don't think we should remove the old `#### Record Stream` Section, and instead this new section should go before it